README.md

```diff
@@ -6,10 +6,39 @@ This is testing having a remote registry for [Singularity HPC](https://github.co
 
 📖️ Read the [documentation](https://singularity-hpc.readthedocs.io/en/latest/) 📖️
 ⭐️ Browse the [container module collection](https://singularityhub.github.io/shpc-registry/) ⭐️
- 
+
+## Contribution rules
```

Comment:
```suggestion
## Contribution
```
They are technically rules, but I think just "Contribution" gets the message across!